tools/respec2html.js

```diff
@@ -30,6 +31,13 @@ if (args.indexOf("-w") !== -1) {
     reportWarnings = true;
 }
 
+if (args.indexOf("--exclude-script") !== -1) {
+    var idx = args.indexOf("--exclude-script");
+    var values = args.splice(idx, 2);
```

Comment:
I've re-used here the approach taken in the two previous argument parsing blocks.